python/mlc_llm/compiler_pass/attach_sampler.py

```diff
@@ -29,7 +29,11 @@ def transform_module(self, mod: IRModule, _ctx: tvm.transform.PassContext) -> IR
             return mod
 
         bb = relax.BlockBuilder(mod)
-        vocab_size = mod["prefill"].ret_struct_info.fields[0].shape[-1]
+        if "prefill" in mod:
+            vocab_size = mod["prefill"].ret_struct_info.fields[0].shape[-1]
+        else:
+            assert "prefill_to_last_hidden_states" in mod
+            vocab_size = mod["prefill_to_last_hidden_states"].ret_struct_info.fields[0].shape[-1]
```

Comment:
Could we have some brief docstring here explaining in what cases `"prefill"` is in `mod` and what cases `"prefill_to_last_hidden_states"` is in `mod`?